client/actions/actions.admin.users.js

```diff
@@ -0,0 +1,79 @@
+import { ActionType } from 'redux-promise-middleware';
+
+export const FETCH_USERS = 'FETCH_USERS';
+export const UPDATE_USERS = 'UPDATE_USERS';
+export const ADD_CHANGE = 'ADD_CHANGE';
+
+export const FETCH_USERS_FULFILLED = `${FETCH_USERS}_${ActionType.Fulfilled}`;
+export const UPDATE_USERS_FULFILLED = `${UPDATE_USERS}_${ActionType.Fulfilled}`;
+
+export const FETCH_USERS_REJECTED = `${FETCH_USERS}_${ActionType.Rejected}`;
+export const UPDATE_USERS_REJECTED = `${UPDATE_USERS}_${ActionType.Rejected}`;
+
+export function fetchUsers() {
+  const payload = new Promise((resolve, reject) => {
+    fetch('/admin/users', {
+      method: 'GET',
+    })
+      .then((response) => response.json())
+      .then((response) => {
+        resolve(response);
+      })
+      .catch((err) => reject(err));
+  });
+
+  return {
+    type: FETCH_USERS,
+    payload,
+  };
+}
+
+export function updateUsers(changes) {
+  const payload = new Promise((resolve) => {
+    fetch('/admin/users', {
+      method: 'PATCH',
```

Comment:
@IgorChaikin Usually `PUT` method used for update operations, also could you please change route for updating orders to `PUT` too